packages/vuepress-theme-odyssey/styles/abstracts/_theming.scss

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020-Present Okta, Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO: Theming stuff is a proof of concept.
```

Comment:
Doesn't look like this file is imported anywhere. Can this be removed from the context of this PR?

Reply:
This is imported in packages/vuepress-theme-odyssey/styles/index.scss:4